plugins/mcreator-localization/help/default/command/type.md

```diff
@@ -0,0 +1,5 @@
+This option allow you to determine the type of the command and how/where it can be used.
+* **Standard**: The command can be used everywhere. This is the normal and ancient behaviour (e.g. `/give`).

```

Comment:
I would avoid the use of ancient as this implies something being old or almost deprecated which this is not.

You should call this default/standard behaviour

Reply:
**Common** would fit too I think 🤔
```suggestion
* **Standard**: The command can be used everywhere. This is the common behavior for most vanilla commands (e.g. `/give`).
```
(If I'm not lying about _most vanilla commands_)